repositories/repository.py

```diff
@@ -1908,6 +2011,30 @@ def update_topic(self, topic_id, topic) -> DM.Topic:
         else:
             raise err.NoValidIdError
 
+    def update_learning_path_learning_element_algorithm(self, topic_id, algorithm_id):
+        entry = self.get_lpath_le_algorithm_by_topic(topic_id)
+        if entry != []:
+            self.session.query(TM.LearningPathLearningElementAlgorithm).filter_by(
+                topic_id=topic_id
+            ).update(
+                {TM.LearningPathLearningElementAlgorithm.algorithm_id: algorithm_id}
+            )
```

Comment:
das formatting is iwi off. hast du mit black shcon formatiert?

Reply:
wird am ende alles formattiert